tiledb/cloud/notebook.py

```diff
@@ -22,6 +22,14 @@
 CHARACTER_ENCODING = "utf-8"
 
 
+class UploadOption(enum.Enum):
```

Comment:
`UploadOption` seems a bit too generic; how about something like `IfExists` (`OnExists`?) Then the docstring can be something like `"""Action to take if the array already exists."""`.

Reply:
Yeap np!

Reply:
I don’t see any new revisions—did you forget to upload? (Certainly something I have done before.)